### PR TITLE
Add WG BYON build pipelines

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

@tmckayus @tumido @goern @durandom

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+sigs: []
```
Comment (durandom) [inline sigs.yaml:1]: I could not really find some docs in the k8s community about what is done with the `sigs.yaml` file. Do you know?

**Review by durandom — APPROVED** [review]
👍 😍

File: sigs.yaml (hunk 1 above)
Comment (goern) [reply sigs.yaml:1]: it is used to create a bunch of .md docs

Comment (pacospace) [reply sigs.yaml:1]: we need to define the sigs chair and tech lead then, like https://github.com/thoth-station/core/blob/master/community/sigs.yaml

Comment (codificat) [reply sigs.yaml:1]: > we need to define the sigs chair and tech lead then

I believe that's the purporse of #2 and https://github.com/goern/open-services-group-community/pull/1

**Review by tumido — APPROVED** [review]
/approve

**Pull request merged**